package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "description": "Description of project here",
   "scripts": {
-    "setup": "yarn && node ./scripts/setup.js",
+    "setup": "yarn && node ./scripts/setup.js && lerna run setup",
```

Comment:
Allow other packages to run setup tasks, such as the mobile package so it can add custom env vars